assets/templates/partials/bulletin/status-header/edition-version-link.tmpl

```diff
@@ -0,0 +1,15 @@
+{{ if .CorrectedPath }}
+  {{/* Detect an old version of a release and link to the latest version */}}
+  <a class="ons-u-nowrap" href="{{ .CorrectedPath }}">
+    {{- localise "StatusLineViewCorrectedVersion" .Language 1 -}}
+  </a>
+{{ else if .LatestRelease }}
+  <a class="ons-u-nowrap" href="{{ .ParentPath }}/previousReleases">
```

Comment:
camel case url?

Reply:
An inherited wart, sadly, that no doubt spills over into Florence and Babbage.

For comparision:

https://github.com/ONSdigital/dp-frontend-articles-controller/blob/0f484b3f35db35bada2ce6f91cf462e66825af0d/assets/templates/sixteens-bulletin.tmpl#L47

Reply:
🤦‍♂️